README.md

```diff
@@ -427,7 +432,7 @@ const gateways = await arIO.getGateways();
   "QGWqtJdLLgm2ehFWiiPzMaoFLD50CnGuzZIPEdoDRGQ": {
     "end": 0,
     "observerWallet": "IPdwa3Mb_9pDD8c2IaJx6aad51Ss-_TfStVwBuhtXMs",
-    "operatorStake": 250000,
+    "operatorStake": 250000, // value in mIO
```

Comment:
Should maybe update this readme example as I think this wouldn't be a valid amount of mIO for operatorStake